### DISCUSSION TITLE
Black frames and video when generating txt2video, no errors in logs

### DISCUSSION THREAD

**Discussion by deroc1** [deforum-art/sd-webui-text2video#202] [Q&A]
In logs it seems normal, prompt and video quality does not matter, i always get a black screen. I also have base settings. I checked another post about this and did not find where is non half-vae mode in text2video settings
![изображение_2023-07-13_091302112](https://github.com/kabachuha/sd-webui-text2video/assets/123573950/e2a0b7e6-262f-4062-93cf-5501fa03a339)
![изображение_2023-07-13_091435554](https://github.com/kabachuha/sd-webui-text2video/assets/123573950/f62cb847-ac54-4fbe-8dc2-4c33620b1d98)
Thanks

**Comment by B34STW4RS** [comment]
I'm confused as to the settings you posted. 3 steps for 5 frames?

Zeroscope was trained with the intent of generating a minimum of 24 frames, it won't work right with a frame count that low.

Also the resolution isn't really supported by the selected model. Have you tried the other vae modes, and actual default settings?
